src/main/webapp/directives/d3Annotation.js

```diff
@@ -57,8 +52,7 @@ angular
                 const wordSpacing = 3.5;
```

Comment:
Can you add a comment to each constant? "oma" and "opa" are still not understandable.